src/mixins/themeable.ts

```diff
@@ -74,25 +74,16 @@ export interface ThemeableMixin extends Evented {
  * Themeable
  */
 export interface Themeable extends ThemeableMixin {
-	baseClasses: BaseClasses;
+	baseClasses: {};
```

Comment:
It didn't work when I used it in my test code. Did you have to case your required `css` as `BaseClasses` ?